lib/fog/azurerm/models/dns/record_sets.rb

```diff
@@ -23,8 +23,10 @@ def all
           load(record_sets)
         end
 
-        def get(identity, type)
-          all.find { |f| f.name == identity && f.type == "Microsoft.Network/dnszones/#{type}" }
+        def get(resource_group, name, zone_name, record_type)
```

Comment:
how end user would be passing record_type argument
- 'M...Network/A/CNAM'
- 'CNAME' 'A'